packages/server/src/db/index.js

```diff
@@ -647,26 +661,76 @@ async function getAgenciesByIds(agencyIds) {
     return result;
 }
 
-async function getAgenciesSubscribedToDigest() {
-    const query = knex
-        .select(
-            'agencies.id',
-            'agencies.name',
-            'agencies.abbreviation',
-            'agencies.code',
+async function getAgenciesSubscribedToDigest(asOf) {
+    const open_date = asOf || moment().subtract(1, 'day').format('YYYY-MM-DD');
+
+    const query = knex.raw(
+        `
+        WITH enabled_codes AS (
+            SELECT
+                a.id AS agency_id,
+                ec.code,
+                CASE WHEN aec.enabled IS NULL THEN
+                    TRUE
+                ELSE
+                    aec.enabled
+                END as enabled
+            FROM
+                eligibility_codes ec
+            CROSS JOIN agencies a
+            LEFT JOIN agency_eligibility_codes aec ON ec.code = aec.code
+                AND a.id = aec.agency_id
+        ),
+        agency_data AS (
+            SELECT
+                a.id,
+                a.name,
+                string_agg(DISTINCT aec.code,
+                    '|') AS codes,
+                string_agg(DISTINCT k.search_term,
+                    '|') AS term,
+                string_agg(DISTINCT u.email,
+                    ' | ') AS emails
```

Comment:
> I noticed that the delimiter for the third string_agg call has whitespace, but the others do not. Was this intentional?

This was intentional because I noticed that `|` character (however unlikely) was a valid character in an email address, so the added space was needed to be compliant.

Reply:
using `array_agg` eliminates these issues! Fixed it in latest commit.